src/components/Header/HeaderUserMenu/HeaderUserMenu.test.js

```diff
@@ -75,8 +75,11 @@ describe('Header user menu', () => {
     // Arrange
     userData.plan = {
       clientManager: {},
+      hasClientManager: false,
+      isSubscribers: true,
```

Comment:
I'm not sure about this flags, with `clientManager` property already here should suffice.